website/docs/r/nodeTemplate.html.markdown

```diff
@@ -255,3 +268,15 @@ Node Template can be imported using the Rancher Node Template ID
 $ terraform import rancher2_node_template.foo <node_template_id>
 ```
 
+## Upgrading to Rancher v2.3.3
+
+Due to [this feature](https://github.com/rancher/rancher/pull/23718) included on Rancher v2.3.3, `rancher2_node_template` are now global scope objects with RBAC arounf them, instead of user scope objects as they were. This means that existing node templates are changing their `id` on upgrade, making that terraform provider will try to recreate them due to it's unable to find them by `id`.
```

Comment:
This means that existing node templates `id` field is changing on upgrade. Because the Terraform provider can not find the old `id` it will try to recreate them.

Reply:
Much better. Updated. :)